src/coreclr/vm/methodtablebuilder.cpp

```diff
@@ -10051,6 +10051,10 @@ void MethodTableBuilder::CheckForSystemTypes()
                     // The Procedure Call Standard for ARM 64-bit (with SVE support) defaults to
                     // 16-byte alignment for __m256.
 
+                    pLayout->m_ManagedLargestAlignmentRequirementOfAllMembers = 16;
+    #elif defined(TARGET_RISCV64)
+                    // The alignment requirement for the fixed length vector shall be equivalent to
```

Comment:
Same fix should be applied to comments in this file.

Reply:
Thank you.